requirements/app/cloud.txt

```diff
@@ -1,3 +1,5 @@
+# WARNING: this file is not used directly by the backend
```

Comment:
If this is the case, then why not merge this with the `base.txt` requirements file?

Reply:
Ah okay! So the intention is to resolve this. Thanks for clarifying